npm-shrinkwrap.json

```diff
@@ -271,9 +271,7 @@
               }
             },
             "commander": {
-              "version": "2.20.3",
-              "resolved": "https://registry.npmjs.org/commander/-/commander-2.20.3.tgz",
```

Comment:
Weird that these entries have been removed for many packages 👀 

I did this locally with node `v10.16.0`. I updated only `@woocommerce/e2e-environment` in `package.json` to `^0.2.2`, then I ran `npm install`.
In my case I didn't get as many removals on `resolved` and `integrity`: https://github.com/Automattic/woocommerce-services/pull/2452/files

It's kinda interesting how these things happen 😅